### DISCUSSION TITLE
Free Hobby Teams

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#6298] [Announcements]
In January, we announced some upcoming account changes. Everyone's personal projects will be moved to a free hobby team. This change makes the transition between hobby and pro plans easier. You can read the original announcement at [vercel.com/changelog/2024-01-account-changes](https://vercel.com/changelog/2024-01-account-changes)

The key benefit of this change is that you no longer need to create a whole new team if you want to [upgrade from hobby to pro](https://vercel.com/docs/accounts/plans/pro#upgrading-to-pro). And you can just as easily [downgrade back to hobby](https://vercel.com/docs/accounts/plans/hobby#downgrading-to-hobby) if you no longer need a pro team. While that's helpful for a lot of people, there are a few differences I have seen with this account update which might be a bit confusing at first. There are a few things to be aware of when your account is updated.

1. You now have a hobby team on your personal account and all of your projects belong to the team instead of to your account. This should mostly look the same, but you'll see `{your name}'s projects` instead of just `{your name}` in the team picker at the top of the page. This new team has all of the same abilities and limits that your hobby account already had. 

2. Your domains are still associated with your **personal account**, so you won't see them listed when you click on the **team** "Domains" tab near the top of the dashboard. Instead, you should click on your profile picture in the top right corner, click "Settings" in the dropdown panel, then click "[Domains](https://vercel.com/account/domains)" on the left side navigation list. From there you should be able to see your domains and move them to your new team using these steps: https://vercel.com/docs/projects/domains/transfer-your-domain

3. If your project is asking for a TXT record to verify your custom domain, but it didn't require this before, you may need to move the domain from your personal account to your new team (see instructions above).

Please feel free to ask questions and share feedback in comments on this post. 🙂

**Comment by franciscoj** [comment]
Just for the record, I wasn't able to either manage or see the DNS entries on my domains until I did this.

The dashboard wasn't showing their DNS entries and I wasn't able to edit or manage them, neither form the dashboard, nor from the API through terraform.

Once I did this everything went back to normal.

**Comment by zowebs** [comment]
Had quite a hard time figuring out where have my hobby domains gone. This post should be associated with at least small announcement on the domains route for new transferred hobby teams, at least on the first-time load 🙏.

Anyway, @amyegan, thank you so much for this post! Saved me some stress..

**Comment by andrei-alves** [comment]
When adding a new domain using my team's menu, I am being redirected to a 404 page that shows a button for "Sign in as a different user". The URL displayed is https://vercel.com/[teamSlug]/[project]/settings/domains. It 

![Captura de tela 2024-06-07 091157](https://github.com/vercel/community/assets/138537998/d90c5e0e-d51a-490a-8c0d-8da2d2b61f13)

**Reply by amyegan** [reply]
Hey @andrei-alves. Thanks for reporting this. 

Can you walk me through the steps that led to the error? I want to make sure I investigate the right workflow.

**Reply by andrei-alves** [reply]
Hi @amyegan! Thank you so much answering!

1. I clicked on the Domains tab on my dashboard;
![image](https://github.com/vercel/community/assets/138537998/b584a812-038a-462f-b7ef-3e58ba29fb36)
2. Then clicked on the Add button;
![image](https://github.com/vercel/community/assets/138537998/3e2f6bbb-6a71-4378-8ab3-007a02187ec8)
3. I select the project to add the new domain, type it, and when I click on Add...
![image](https://github.com/vercel/community/assets/138537998/56fe021b-ed93-4929-a709-a2aa94d7bfff)
4. I am sent to this page:
![Captura de tela 2024-06-07 091157](https://github.com/vercel/community/assets/138537998/d63cfaf7-0e1a-41f0-80bd-beb715ca99f0)
5. But when I return to my dashboard, I see the new domain on the projects card:
![image](https://github.com/vercel/community/assets/138537998/8b6be284-48c3-41c7-8bfc-064ac00014e5)

**Reply by amyegan** [reply]
Thanks for giving me the steps! I see the same error on my Pro and Hobby teams. 

I opened a ticket for our domains team to look into the cause of this broken redirect and get it fixed.

**Reply by andrei-alves** [reply]
Thank you so much @amyegan! ❤

**Reply by amyegan** [reply]
This is fixed now. Thanks again for letting us know!

**Reply by andrei-alves** [reply]
Thank you so much again for your promptness, @amyegan! 🤩💪

**Comment by awmartin** [comment]
I'm quite confused by the Hobby Plan. I'm a new to Vercel and am evaluating it for hosting mutliple projects, but it seems I can only create one project? Whenever I attempt to add another project, it asks me to create a team and upgrade to Pro. I thought we could host more than one project on the Hobby Plan, but apparently that's not the case?

**Reply by amyegan** [reply]
Hey @awmartin! You can have multiple projects on Hobby plans (see [vercel.com/docs/accounts/plans/hobby](https://vercel.com/docs/accounts/plans/hobby) for details). I think you might have run into a different limit.

Vercel does not allow Hobby projects connected to Git repos owned by a Git organization (e.g. GitHub Organization, GitLab group, Bitbucket team). That is a common reason that people see a prompt instructing them to upgrade to a Pro plan.

https://vercel.com/docs/limits/overview#connecting-a-project-to-a-git-repository

**Reply by awmartin** [reply]
Thanks for this. But the article doesn't quite make sense to me. (Also the restriction seems rather silly. Anyone can create an org.) It also refers to "Personal" accounts; is that the same as Hobby?

The source for the project I have currently have deployed is indeed from a GitHub organization. How was I able to do this if they're not allowed in the first place?

In any case, it seems this limit should be posted on a description of Hobby Plans, so we're aware.

**Reply by amyegan** [reply]
Thanks for responding. Let's continue our conversation in https://github.com/vercel/community/discussions/7246#discussioncomment-9832972

**Comment by fintdevla** [comment]
I've a problem related to this one. I can't enable Vercel DNS on my domain, fint.app, as I was trying to use wildcard domains.

[a comment by an automated account is not shown]
